Index: Fiak/yaak/src-web/components/core/Link.tsx
===================================================================
--- Fiak.orig/yaak/src-web/components/core/Link.tsx
+++ Fiak/yaak/src-web/components/core/Link.tsx
@@ -1,7 +1,6 @@
 import { Link as RouterLink } from '@tanstack/react-router';
 import classNames from 'classnames';
 import type { HTMLAttributes } from 'react';
-import { appInfo } from '../../lib/appInfo';
 import { Icon } from './Icon';
 
 interface Props extends HTMLAttributes<HTMLAnchorElement> {
@@ -20,19 +19,12 @@ export function Link({ href, children, n
   );
 
   if (isExternal) {
-    const isYaakLink = href.startsWith('https://yaak.app');
-    let finalHref = href;
-    if (isYaakLink) {
-      const url = new URL(href);
-      url.searchParams.set('ref', appInfo.identifier);
-      finalHref = url.toString();
-    }
     return (
       // eslint-disable-next-line react/jsx-no-target-blank
       <a
-        href={finalHref}
+        href={href}
         target="_blank"
-        rel={isYaakLink ? undefined : 'noopener noreferrer'}
+        rel="noopener noreferrer"
         onClick={(e) => e.preventDefault()}
         className={className}
         {...other}
